src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(number) {
+      calculator.result += number;
+    },
+
+    subtract(number) {
+      calculator.result -= number;
+    },
+
+    divide(number) {
+      calculator.result /= number;
+    },
+
+    multiply(number) {
+      calculator.result *= number;
+    },
+
+    reset() {
+      calculator.result = 0;
```

Comment:
Use 'this.result' instead of 'calculator.result' to avoid potential scoping issues.